content/id/guides/configuration-glossary/configuration-telemetry.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Properti telemetry'
+description: 'Nuxt.js mengumpulkan data-data telemetri secara anonim mengenai penggunaan Nuxt.js secara umum. Hal tersebut membantu kami untuk mengukur penggunaan fitur dan kustomisasi dalam Nuxt.js secara akurat pada seluruh pengguna Nuxt.js.'
+menu: telemetry
+category: configuration-glossary
+position: 30
+---
+
+## Properti telemetry
```

Comment:
Saya kira `telemery` disini bisa disamakan seperti yang bawah menjadi `telemetri`

Reply:
Bagian tersebut tidak saya terjemahkan karena `telemetry` yang dimaksud pada konteks ini adalah bagian dari properti sebuah objek (mungkin juga saya salah memahami konteksnya).

Dalam bahasa Inggris, nilai dari `title` adalah: 'The telemetry property'